playbooks/roles/nghttpx/files/snakeoil.pem

```diff
@@ -0,0 +1,46 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
What I'm saying is that you can generate this private key & certificate on the fly versus hardcoding them.